code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/ServiceDiscoveryService.java

```diff
@@ -39,5 +39,4 @@
 
     List<Integer> getServiceInstanceUsedOrderIds(Service service);
 
```

Comment:
Also, sadly, forgot to remove this after having DeploymentManagerImpl retrieve global services from expMapDao instead of sdSvc